pipeline.Dockerfile

```diff
@@ -1,47 +1,50 @@
 FROM python:3.6
 
 RUN apt-get update && apt-get install -y software-properties-common \
-	&& apt-get install -y build-essential libssl-dev libffi-dev zlib1g-dev libjpeg-dev git \
+	&& apt-get install -y build-essential libssl-dev libffi-dev zlib1g-dev libjpeg-dev git jq=1.5+dfsg-1.3\
 	&& apt-get clean all
 
-RUN pip install --disable-pip-version-check --upgrade pip==18.1 pipenv==2018.10.13
+# Install Kops
+ENV KOPS_VERSION=1.10.0
+ADD https://github.com/kubernetes/kops/releases/download/${KOPS_VERSION}/kops-linux-amd64 /usr/local/bin/kops
+RUN chmod a+x /usr/local/bin/kops
 
-# Install requirements for legion package
-ADD legion/requirements/Pipfile /src/requirements/legion/Pipfile
-ADD legion/requirements/Pipfile.lock /src/requirements/legion/Pipfile.lock
-RUN cd /src/requirements/legion && pipenv install --system --dev
+# Install kubectl
+ENV KUBECTL_VERSION=v1.10.6
+ADD https://storage.googleapis.com/kubernetes-release/release/${KUBECTL_VERSION}/bin/linux/amd64/kubectl /usr/local/bin/kubectl
+RUN chmod a+x /usr/local/bin/kubectl
 
-# Install requirements for legion_test package
-ADD legion_test/requirements/Pipfile /src/requirements/legion_test/Pipfile
-ADD legion_test/requirements/Pipfile.lock /src/requirements/legion_test/Pipfile.lock
-RUN cd /src/requirements/legion_test && pipenv install --system --dev
+# Install Helm
+ENV HELM_VERSION=v2.10.0
+ADD https://kubernetes-helm.storage.googleapis.com/helm-${HELM_VERSION}-linux-amd64.tar.gz /tmp/helm/
+RUN tar xzf /tmp/helm/helm-${HELM_VERSION}-linux-amd64.tar.gz -C /tmp/helm && \
+    mv /tmp/helm/linux-amd64/helm /usr/local/bin/helm && rm -rf /tmp/helm
 
-# Install requirements for legion_airflow package
-ADD legion_airflow/requirements/Pipfile /src/requirements/legion_airflow/Pipfile
-ADD legion_airflow/requirements/Pipfile.lock /src/requirements/legion_airflow/Pipfile.lock
-RUN cd /src/requirements/legion_airflow && pipenv install --system --dev
+RUN pip install --disable-pip-version-check --upgrade pip==18.1 pipenv==2018.10.13 
 
 # Install additional tools for build purposes
-RUN pip install Sphinx==1.8.0 sphinx_rtd_theme==0.4.1 sphinx-autobuild==0.7.1 recommonmark==0.4.0 twine==1.11.0 ansible==2.6.4 awscli==1.16.19
+RUN pip install Sphinx==1.8.0 sphinx_rtd_theme==0.4.1 sphinx-autobuild==0.7.1 \
+  recommonmark==0.4.0 twine==1.11.0 awscli==1.16.85 ansible==2.7.2 yq==2.7.1
 
 # Add sources
-
-
 ADD legion /src/legion
+RUN cd /src/legion/requirements && pipenv install --system --dev
```

Comment:
Please move requirements installation back to previous layers (for caching purposes)

Reply:
done